modules/gateways/blockonomics/blockonomics.php

```diff
@@ -521,6 +521,13 @@ public function processOrderHash($order_hash, $blockonomics_currency)
         }
         // Process a new order for the id and blockonomics currency
         $new_order = $this->createNewCryptoOrder($order_info, $blockonomics_currency);
+        $log_data = array(
+            'order_id' => $new_order->id_order,
```

Comment:
We save this as id_order, however this matches a WHMCS invoice id.

Reply:
Updated